src/main/java/policycompass/fcmmanager/controllers/FCMModels.java

```diff
@@ -753,4 +758,81 @@ public static FCMModel LoadData() {
 		session.close();
 		return model;
 	}
+
+	public static int isAdminUser(String userPath,String userToken)
+	{
+		int returnValue=-1;
+		String charset = "UTF-8";
+		String requestURL = "https://adhocracy-frontend-stage.policycompass.eu/api";
+		String requestURLGod = "https://adhocracy-frontend-stage.policycompass.eu/api/principals/groups/gods/";
+
+		try {
+
+			URL url = new URL(requestURL);
+			HttpURLConnection httpConn = (HttpURLConnection) url.openConnection();
+			httpConn.setUseCaches(false);
+			httpConn.setDoOutput(true); // indicates POST method
+			httpConn.setDoInput(true);
+
+			httpConn.setRequestProperty("X-User-Path", userPath);
+			httpConn.setRequestProperty("X-User-Token", userToken);
+
+			List<String> response = new ArrayList<String>();
+			int status = httpConn.getResponseCode();
+			if (status == HttpURLConnection.HTTP_OK) {
+				BufferedReader reader = new BufferedReader(new InputStreamReader(
+						httpConn.getInputStream()));
+				String line = null;
+				while ((line = reader.readLine()) != null) {
+					response.add(line);
+				}
+				reader.close();
+				httpConn.disconnect();
+
+				if(response.toString().toLowerCase().contains("error")){
```

Comment:
Don't call toString on a List to then perform a string comparison on the result for validation. You are not guaranteed to get the full list content there.

Use a `StringBuilder`.